### PR TITLE
Search Engine Optimization

### PR DESCRIPTION
**Pull request by AlphaT3ch**
See [WooRank](https://www.woorank.com/en/www/botwillacceptanything.com) and [Validator](https://validator.nu/?doc=http%3A%2F%2Fbotwillacceptanything.com).

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 c77a9152a0635a2ee33ca428c0d7c77ea1ab7d1f

File: data/views/layouts/main.hbs [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-<html>
+<html lang="en">
     <head>
         <link rel="icon" type="image/ico" href="/favicon.ico">
-        <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="Content-Language" content="en" />
+        <meta http-equiv="content-type" content="text/html; utf-8">
```
Comment (SpenserJ) [inline data/views/layouts/main.hbs]: You still have a Content Type meta in here twice (also line 5)

File: data/views/layouts/main.hbs [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
         <title>Bot Will Accept Anything</title>
+        <meta name="Description" content="Bot Will Accept Anything - The project where anything goes, as long as the code allows it">
         <meta name="author" content="GitHub Commitors">
+        <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
```
Comment (SpenserJ) [inline data/views/layouts/main.hbs]: You still have the HTML declaration in here twice (line 2). Remove this one.

**Comment by SpenserJ** [comment]
:-1: for the reasons listed in comments

**Comment by AlphaT3ch** [comment]
@SpenserJ: You still have a Content Type meta in here twice (also line 5)...

Uh, actually one is content type and the other one is content language... They are different

**Comment by SpenserJ** [comment]
@AlphaT3ch Did you look at line 5, like I mentioned? [Link](https://github.com/botwillacceptanything/botwillacceptanything/pull/454/files#diff-d43d80b28963d90614bfb8b64e621cfcR5)

**Pull request closed without merging**